cluster-provision/k8s/base-image

```diff
@@ -1 +1 @@
-quay.io/kubevirtci/centos9:2504010400-a291de27
+quay.io/kubevirtci/centos9:2504020907-d57fd6e1
```

Comment:
Why is this change necessay? 
Does this image include Istio binary, or include preloaded Istio images?

Reply:
Thanks for explaining, does this image also include pre-loaded images? 
If not, maybe it worth to mirror the binary somewhere outside the node image, that will save provisioning new node images.

Reply:
No the base image doesn't include any preloaded images. 

It allows us to pin as many of the centos stream deps as we can in order to avoid stream updates breaking kubevirtci delivery into kubevirt.

Reply:
Istio binary is downloaded directly from Istio Github repo release, and not thorough CentOS repositories.

The fact we need to provision the node image on every Istio version change is cumbersome :/ 
If istio binary would be mirrored outside the node image it could make Istio bump process much more efficient and easy :smiling_face_with_tear: